docs/deployment.md

```diff
@@ -0,0 +1,90 @@
+# Deployments
+
+## Next.js app
+
+The Next.js app can be deployed as a static HTML export or as a Node.js server, depending on your needs. The [Next.js deployment documentation](https://nextjs.org/docs/deployment) provides more information on how to deploy your application.
+
+## Storybook
+
+Storybook can be deployed as a static HTML export. The [Storybook deployment documentation](https://storybook.js.org/docs/react/sharing/publish-storybook) provides more information on deployment options.
+
+### GitHub Pages
+
+[GitHub Pages](https://pages.github.com/) may be a good option for hosting Storybook if you only need one environment to represent the latest state of your main branch.
+
+**Steps to setup GitHub Pages**
+
+1. Add a workflow to automatically deploy Storybook: `.github/workflows/deploy-storybook.yml` (see example below)
+1. Enable the "GitHub Actions" source in your rep settings (`Settings > Pages > Source`)
```

Comment:
wasn't sure what these instructions meant. does the user need to select a source?
<img width="762" alt="image" src="https://user-images.githubusercontent.com/447859/204401247-9ab62e5c-d96b-4809-959a-969a9ec38f1f.png">

should it be the main branch or a different branch? seems like it'd be simpler to be prescriptive.

Reply:
@lorenyu Is this phrasing clearer?

> Select "GitHub Actions" from the **Source** dropdown field on the "Pages" tab in your repo settings (`Settings > Pages`).

Reply:
ah yeah i see what you mean now thanks